packages/db/src/config.ts

```diff
@@ -33,24 +34,39 @@ export const getSecretConnectionConfig = async (
       throw new Error(`AWS Secret ${SecretId} is missing required key '${key}'`);
     }
   });
+  const rejectUnauthorized = dbAccessMeta.rejectUnauthorized !== 'false' && dbAccessMeta.rejectUnauthorized !== false;
+  const useSsl = dbAccessMeta.disableSSL === 'true' || dbAccessMeta.disableSSL === true;
   return {
+    database: dbAccessMeta.database,
     host: dbAccessMeta.host,
-    user: dbAccessMeta.username,
     password: dbAccessMeta.password,
-    database: dbAccessMeta.database,
     port: dbAccessMeta.port ?? 5432,
+    ssl: useSsl ? undefined : { rejectUnauthorized },
```

Comment:
So `useSsl` is `true` if `dbAccessMeta.disableSSL === 'true'`? 🤯 

Then if `useSsl` isn't explicitly `true`, we default to `rejectUnauthorized`. That I think makes sense but the first part is causing an aneurysm.

Reply:
Oof, that's .... just bad naming.  Try: 0f0e36e0219

Reply:
Thanks 😄